dist/bottle.js

```diff
@@ -1,7 +1,7 @@
 ;(function(undefined) {
     'use strict';
     /**
-     * BottleJS v1.3.0 - 2016-04-29
+     * BottleJS v1.3.0 - 2016-07-06
```

Comment:
@nzack Can you remove the dist generated files from your PR?  I'll generate them when a new bottle version is released.